test/services/account/index.test.js

```diff
@@ -169,89 +170,39 @@ describe('Account Service', () => {
 		});
 
 		it('should fail to patch the password if it does not meet requirements', async () => {
+			const user = await testObjects.createTestUser();
 			const accountDetails = {
 				username: 'some@user.de',
 				password: 'ca4t9fsfr3dsd',
-				userId: new ObjectId(),
+				userId: user._id,
 			};
 			const account = await accountService.create(accountDetails);
 			try {
-				await accountService.patch('0000d213816abba584714c0a', { password: '1234' });
+				const params = await generateRequestParams(accountDetails);
+				await accountService.patch(account.id, { password: '1234' }, params);
 				throw new Error('This should not happen.');
 			} catch (exception) {
 				assert.equal(exception.message, 'Dein Passwort stimmt mit dem Pattern nicht überein.');
 			} finally {
-				await accountService.remove(account._id);
+				await accountService.remove(account.id);
+				await userService.remove(user.id);
 			}
 		});
 
 		it('should fail on verification mismatch', async () => {
-			const user = await testObjects.createTestUser();
+			const user = await testObjects.createTestUser({ firstLogin: true });
 			const accountDetails = {
 				username: 'some@user.de',
 				password: 'ca4t9fsfr3dsd',
 				userId: user._id,
 			};
 			const account = await accountService.create(accountDetails);
 			try {
+				const params = await generateRequestParams(accountDetails);
 				await accountService.patch(account._id, {
 					password: 'Schul&Cluedo76 ',
-					password_verification: accountDetails.password,
-				}, {
-					account: {
-						userId: user._id,
-						password: '$2a$10$CHN6Qs2Igbn.s4BengUOfu9.0qVuy0uyTrzDDJszw9e1lBZwUFqeq',
-					},
-				});
-				throw new Error('This should not happen.');
-			} catch (exception) {
-				assert.equal(exception.message, 'Dein Passwort ist nicht korrekt!');
-			} finally {
-				await accountService.remove(account._id);
-			}
-		});
-
-		it('should fail to patch the password if the password verfication is wrong', async () => {
-			const userObject = {
-				firstName: 'Max',
-				lastName: 'Mustermann',
-				email: `max${Date.now()}@mustermann.de`,
-				schoolId: '0000d186816abba584714c5f',
-				preferences: {
-					firstLogin: true,
-				},
-			};
```

Comment:
Why are you removing a test case?

Reply:
This test did duplication both check if the test fails when the password verification is wrong. Doesn't make sense to test twice the same.

Reply:
👍